packages/pluggableWidgets/line-chart-native/src/components/LineChart.tsx

```diff
@@ -87,63 +110,63 @@ export function LineChart(props: LineChartProps): ReactElement | null {
-                                padding={style.chart?.padding}
-                                scale={
-                                    dataTypesResult
-                                        ? {
-                                              x: dataTypesResult.x === "number" ? "linear" : "time",
-                                              y: dataTypesResult.y === "number" ? "linear" : "time"
-                                          }
-                                        : undefined
-                                }
-                                style={style.chart}
-                            >
-                                <VictoryAxis
-                                    style={style.xAxis}
-                                    axisLabelComponent={
-                                        xAxisLabel ? (
-                                            <VictoryLabel
-                                                dx={style.xAxis?.axisLabel?.horizontalOffset}
-                                                dy={style.xAxis?.axisLabel?.verticalOffset}
-                                            />
-                                        ) : (
-                                            undefined
-                                        )
-                                    }
-                                    label={xAxisLabel}
-                                    orientation={"bottom"}
-                                    {...(firstSeries?.xFormatter ? { tickFormat: firstSeries.xFormatter } : undefined)}
-                                />
-                                <VictoryAxis
-                                    dependentAxis
-                                    style={style.yAxis}
-                                    axisLabelComponent={
-                                        yAxisLabel ? (
-                                            <VictoryLabel
-                                                dx={style.yAxis?.axisLabel?.verticalOffset}
-                                                dy={style.yAxis?.axisLabel?.horizontalOffset}
-                                            />
-                                        ) : (
-                                            undefined
-                                        )
-                                    }
-                                    label={yAxisLabel}
-                                    orientation={"left"}
-                                    {...(firstSeries.yFormatter ? { tickFormat: firstSeries.yFormatter } : undefined)}
-                                />
-                                {chartLines}
-                            </VictoryChart>
+                <View style={style.chart}>
+                    <View style={style.gridAndLabelsRow}>
+                        {yAxisLabel && yAxisLabelRelativePositionGrid === "top" ? (
+                            <Text style={yAxisLabelStyle}>{yAxisLabel}</Text>
```

Comment:
What do you think about creating a variable and referencing this here and in the next usage, line 120?

Reply:
That's better indeed. Changed it.